plugin/type_uuid/item_uuidfunc.cc

```diff
@@ -16,6 +16,10 @@
 #define MYSQL_SERVER
 #include "mariadb.h"
 #include "item_uuidfunc.h"
+#include <myisampack.h>
```

Comment:
myisam is some storage engine, i.e. a plugin, ideally. It's not correct to use its library in general.